docs/howto-openshift-crc.md

```diff
@@ -0,0 +1,258 @@
+# Deploy samba-operator over OpenShift-CRC
+
+The following document describe how to deploy samba-operator and create
+SMB shares over OpenShift Container Platform 4 using
+[crc](https://crc.dev/crc/). This mode of operation mainly targeted at
+running on developers' Linux desktops and requires
+[minimal system resources](https://crc.dev/crc/#minimum-system-requirements-hardware_gsg).
+It also requires virtualization enabled on you local machine.
+
+The instructions in this document where tested with the following
+settings:
+
+```sh
+  $ uname -msr
+  Linux 6.1.18-100.fc36.x86_64 x86_64
+  $ crc version
+  CRC version: 2.15.0+cc05160
+  OpenShift version: 4.12.5
+  Podman version: 4.3.1
+  $ qemu-kvm --version
+  QEMU emulator version 6.2.0 (qemu-6.2.0-17.fc36)
+```
+
+## Setup OpenShift CRC cluster
+Download openshift's crc to your local Linux machine using
+[crc installing instructions](https://crc.dev/crc/#installing_gsg), and
+place the `crc` executable within your `PATH`. Make sure that you have
+an updated pull-secret stored within a local file (`pull-secret.txt`).
+Start a new crc with the following command (may take few minutes):
+
+```sh
+  $ crc start -p ./pull-secret.txt
```

Comment:
Don't we need to mention `crc setup` step before running `crc start`? I hit the following error:
```
$ crc start -p pull-secret.json 
file '/home/anoopcs/.crc/cache/crc_libvirt_4.12.5_amd64.crcbundle' does not exist
```

Reply:
It is hidden in the assumptions that you first followed the instructions at https://crc.dev/crc/#installing_gsg. That said, you have a valid point: `crc setup` should be part of this doc as well. Will fix.